docs/book/src/clusterctl/commands/upgrade.md

```diff
@@ -77,19 +77,10 @@ clusterctl upgrade apply \
 
 <aside class="note warning">
 
-<h1>Clusterctl upgrade test coverage</h1>
+<h1>Skip upgrades</h1>
```

Comment:
Let's link to the place where we describe the skip upgrade policy for core CAPI / CABPK / KCP